addons/mail/static/src/core/common/chat_hub.xml

```diff
@@ -2,28 +2,28 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.ChatHub">
-    <div class="o-mail-ChatHub" t-if="isShown or ui.isSmall">
+    <div class="o-mail-ChatHub d-print-none" t-if="isShown or ui.isSmall">
```

Comment:
Don't we need some/most of those fixes in stable too?

Reply:
At least some, but I'd rather backporting them after this PR is merged.